packages/domain/lib/entities/movies_response_entity.dart

```diff
@@ -1,9 +1,11 @@
 class MoviesResponseEntity {
   List<dynamic>? movies;
   Map<String, List<String>> headers;
+  String? errorMessage;
```

Comment:
I don't think it is the best solution. better to create an exception class in domain and throw its object in data layer